### ISSUE TITLE
How to add new language?

### ISSUE THREAD

**Issue by tiborepcek** [FreshRSS/FreshRSS#2499]
I tray to add new language (Slovak), which [i translated](https://github.com/FreshRSS/FreshRSS/pull/2497) few days ago.

This is what i get in Configuration - Display:

![freshrss-lang_name](https://user-images.githubusercontent.com/10828749/63362911-e52b0280-c372-11e9-8c20-f6feeb0ed6f3.png)

When i choose `gen.lang.sk`, FreshRSS refuse to work without errors in log file.

Any ideas, how to correctly install new language into FreshRSS?

Thank you.

**Comment by Frenzie** [comment]
Not 100% sure otoh but I imagine that it would be fine if you added an `sk` in the English language file. Unless you mean it doesn't work beyond just this minor missing string?

https://github.com/FreshRSS/FreshRSS/blob/d09826c145bd6276f8a0e85a87dccefb55227ca1/app/i18n/en/gen.php#L117-L132

**Comment by tiborepcek** [comment]
Now, instead of gen.lang.sk it shows correctly the name of the language (Slovenčina), but when i choose the new language, FreshRSS still does not work. My web browser shows `server error 500` without FreshRSS error in log.txt.

**Comment by Frenzie** [comment]
Could be as simple as wrong owner/permissions or something? I'm a bit surprised that there wouldn't be anything in the error log; did you also check your server's error log?

**Comment by tiborepcek** [comment]
I checked `error_log` on my hosting server and i did find the errors. There were missing quotation marks in files `index.php` and `gen.php` in folder `sk`. Now everything works just fine. New PR is on the way. :)

**Comment by Frenzie** [comment]
Oh, but it says so right in the Travis builds! That's what they're there for. ;-)

```
$ if [ -s php-l-results ]; then cat php-l-results; exit 1; fi

PHP Parse error:  syntax error, unexpected 'remove' (T_STRING), expecting ')' in ./app/i18n/sk/gen.php on line 18

xargs: php: exited with status 255; aborting

PHP Parse error:  syntax error, unexpected 'mark_all_read' (T_STRING), expecting ')' in ./app/i18n/sk/index.php on line 40

xargs: php: exited with status 255; aborting
```

**Comment by tiborepcek** [comment]
Sorry, my fault...

**Comment by Frenzie** [comment]
No problem at all. Now you know. :-)

The CI performs some basic sanity checks (like syntax), so if it errors we check why before merging. Generally that means we don't merge until it's all green checkmarks.
